src/util/input.rs

```diff
@@ -54,21 +54,27 @@ pub fn process_mouse(event: MouseEvent, context: &mut AppContext, backend: &mut
         MouseEvent::Press(MouseButton::Left, x, y)
             if y > layout_rect[1].y && y <= layout_rect[1].y + layout_rect[1].height =>
         {
-            if x < layout_rect[1].x {
-                if let Some(dirlist) = context.tab_context_ref().curr_tab_ref().curr_list_ref() {
```

Comment:
The actual bug was in this line by using `curr_list_ref` instead of `parent_list_ref`.